src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -122,6 +135,14 @@ internal MsQuicStream(SafeMsQuicConnectionHandle connection, QUIC_STREAM_OPEN_FL
                 _stateHandle.Free();
                 throw;
             }
+
+            if (NetEventSource.Log.IsEnabled())
+            {
+                NetEventSource.Info(
+                    _state,
+                    $"[Stream#{_state.GetHashCode()}] outbound {(unidirectional ? "uni" : "bi")}directional stream created " +
```

Comment:
```suggestion
                    $"[Stream#{_state.GetHashCode()}] outbound {(_canRead ? "bi" : "uni")}directional stream created " +
```
Just a suggestion, the `bool unidirectional` variable is not strictly needed, but if you think it improves readability, please keep it.

Reply:
In a method nearby it was a stream in a different direction, so it felt weird to write `{(_canRead ? "bi" : "uni")}` here and `{(_canWrite ? "bi" : "uni")}` there. But I don't have a strong opinion here